hack/yq-fixup-yamls.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# use yq to reformat yaml files to fit yamllint's syntax validity checks
+#
+# usage:
+#   yq-fixup-yamls.sh file.yaml
+#   yq-fixup-yamls.sh dirpath
+#
+set -e
+YQ=${YQ:-$(command -v yq)}
+
+_yaml_fixup_file() {
+	yaml="$1"
+
+	${YQ} eval "${yaml}" > "${yaml}".tmp
+	mv "${yaml}".tmp "${yaml}"
```

Comment:
There is an inplace option in yq if you prefer that approach.

Reply:
Missed this option... using '--inplace' feels better (avoid leaving '*.tmp' files behind). Will change.